### PR TITLE
Data cleaning

### PR DESCRIPTION
**Pull request by marieke6**
Added notes about data cleaning methods and exploration

Issue fix: #114

### CODE REVIEW

**Review by rogov-dvp — APPROVED** [review]
Nice job Merieke :)

File: docs/data_preprocessing/data_cleaning.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Data Prweprocessing
```
Comment (emedema) [inline docs/data_preprocessing/data_cleaning.md:1]: Preprocessing not prweprocessing

**Review by emedema — CHANGES_REQUESTED** [review]

File: docs/data_preprocessing/data_cleaning.md (hunk 1 above)
Comment (marieke6) [reply docs/data_preprocessing/data_cleaning.md:1]: Ahh good catch thanks!

**Review by emedema — APPROVED** [review]
looks good to me!

**Pull request merged**